DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UITextViewRecorder.swift

```diff
@@ -40,7 +40,7 @@ internal struct UITextViewRecorder: NodeRecorder {
             contentRect: CGRect(origin: textView.contentOffset, size: textView.contentSize)
         )
         let node = Node(viewAttributes: attributes, wireframesBuilder: builder)
-        return SpecificElement(subtreeStrategy: .record, nodes: [node])
+        return SpecificElement(subtreeStrategy: .ignore, nodes: [node])
```

Comment:
question
does this mean previously we were recording the new lines (carriage) but not anymore?

I still see in the snapshots the number of words displayed in the actual app vs wireframe differs. Is there a plan to fix it or we are ignoring that level of granularity.